.github/workflows/publish-release.yml

```diff
@@ -1,33 +1,27 @@
-# Source: https://github.com/release-drafter/release-drafter/blob/master/.github/workflows/release.yml
 ---
 
-name: Release
+name: GitHub Tag Update
+
 on:
-  push:
-    tags:
-      - v*.*.*
+  release:
+    types: [ published, edited ]
+
+permissions:
+  contents: read
 
 jobs:
-  release:
+  tag-update:
     runs-on: ubuntu-latest
 
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
+    permissions:
+      contents: write
 
-      - name: Version
-        id: version
-        run: |
-          tag=${GITHUB_REF/refs\/tags\//}
-          version=${tag#v}
-          major=${version%%.*}
-          echo "::set-output name=tag::${tag}"
-          echo "::set-output name=version::${version}"
-          echo "::set-output name=major::${major}"
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v3
 
-      - name: force update major tag
-        run: |
-          git tag v${{ steps.version.outputs.major }} ${{ steps.version.outputs.tag }} -f
-          git push origin refs/tags/v${{ steps.version.outputs.major }} -f
+      - name: Update tag
+        uses: Actions-R-Us/actions-tagger@v2
+        with:
+          publish_latest_tag: true
```

Comment:
@zmiklank But If you decide that having a `latest` tag isn't beneficial for your action, [`Actions-R-Us/actions-tagger`](https://github.com/Actions-R-Us/actions-tagger#readme) allows you to disable `latest` tag.

Reply:
Yes, I think it is better for this action to not enable the latest tag.

Reply:
Done